tests/end2end/features/thumbnail/test_thumbnaillazy.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Lazy load thumbnails
+
+    Scenario: Eager loading (default behavior)
+        Given I open 10 images without leading zeros in their name
+        And I enter thumbnail mode
+        Then there should be 10 rendered thumbnails
+        When I run 5scroll right
+        Then there should be 10 rendered thumbnails
+        And the first index should be 0
+        And the last index should be 9
+
+    Scenario: Only display the selected thumbnail
+        Given I open 10 images without leading zeros in their name
+        And I enter thumbnail mode
+        Then there should be 10 rendered thumbnails
+        When I run set thumbnail.load_behind 0
+        When I run set thumbnail.load_ahead 0
```

Comment:
You could consider replacing multiple subsequent `When` statements with `When`, `And`, `And`, ... This is more consistent with the rest of the tests, but not a deal-breaker if you prefer it as-is.